integration/kubernetes/helm-chart/alluxio/templates/_helpers.tpl

```diff
@@ -160,17 +160,18 @@ resources:
 {{- define "alluxio.worker.tieredstoreVolumeMounts" -}}
   {{- if .Values.tieredstore.levels }}
     {{- range .Values.tieredstore.levels }}
+      {{- /* The mediumtype can have multiple parts like MEM,SSD */}}
       {{- if .mediumtype }}
+        {{- /* Mount each part */}}
         {{- if contains "," .mediumtype }}
-{{- $type := .type }}
-{{- $path := .path }}
-{{- $split := split "," .mediumtype }}
+          {{- $type := .type }}
+          {{- $path := .path }}
+          {{- $split := split "," .mediumtype }}
           {{- range $key, $val := $split }}
-            {{- if eq $type "hostPath"}}
-            - mountPath:  {{ index ($path | split ",") $key }}
-              name: {{ $val | lower }}-{{ $key | replace "_" "" }}
-            {{- end}}
+            - mountPath: {{ index ($path | split ",") $key }}
+              name: {{ $val | lower }}-{{ replace $key "_" "" }}
```

Comment:
This is a bug. Before only `hostPath` is added to Volume mounts when the `mediumtype` has multiple parts.